dev/deps/spark-deps-hadoop-2.7-hive-2.3

```diff
@@ -43,7 +43,6 @@ commons-configuration/1.6//commons-configuration-1.6.jar
 commons-crypto/1.1.0//commons-crypto-1.1.0.jar
 commons-dbcp/1.4//commons-dbcp-1.4.jar
 commons-digester/1.8//commons-digester-1.8.jar
-commons-httpclient/3.1//commons-httpclient-3.1.jar
```

Comment:
Could you try `dev/test-dependencies.sh --replace-manifest` and find which one is using this still?

Reply:
I ran the above command and it seems `spark-deps-hadoop-2.7-hive-2.3` is still using it.

Reply:
I generated the dependency tree for hadoop-2.7 profile, Hadoop-client lists it as a compile dependency
```
+- org.apache.hadoop:hadoop-client:jar:2.7.4:compile
[INFO] |  +- org.apache.hadoop:hadoop-common:jar:2.7.4:compile
[INFO] |  |  +- commons-cli:commons-cli:jar:1.2:compile
[INFO] |  |  +- xmlenc:xmlenc:jar:0.52:compile
[INFO] |  |  +- commons-httpclient:commons-httpclient:jar:3.1:compile
```

Reply:
Done.